src/ui/components/CommandPalette/CommandButton.tsx

```diff
@@ -25,21 +29,18 @@ function CommandButton({ command, executeCommand, active }: CommandButtonProps)
     <button
       key={label}
       onClick={onClick}
-      className={`${bgColors} flex justify-between px-6 py-2 transition`}
+      className={cx(
```

Comment:
Small nit: any reason for the cx instead of the default `classnames`? Only bringing it up since we use `cx` for context, but that's something that doesn't really make a whole lot of sense in replay land so will probably be done away with soon.

Reply:
Multiple projects that used `classnames` library imported it as `cx` and nowadays I don't even think twice about it. No special reason, just shorter than "classnames", I guess.

It seems I'm not the only one following this "convention" based on the search through the codebase for "cx" that I did just now.